packages/core/src/endowmentsToolkit.js

```diff
@@ -80,6 +88,16 @@ function endowmentsToolkit({
         }
         // write access handled elsewhere
         if (packagePolicyValue === 'write') {
+          if (!handleGlobalWrite) {
+            return
+          }
+          if (pathParts.length > 1) {
+            throw new Error(
+              `LavaMoat - write access is only allowed at the top level, saw "${path}"`
+            )
+          }
+          allowedWriteFields.add(path)
+          whitelistedReads.push(path)
```

Comment:
I could let it fall through to the push below, but there's a check for policy field being explicitly `true` not any other accidental value. It's safer to have a clear carve-out for `'write'`